index.html

```diff
@@ -0,0 +1,1546 @@
+        <div class="mag-img-div">
+          <img class="w-100 h-100 mag-img" src="./images/ft3.jpg" alt="Card image cap" />
+        </div>
+        <div class="mag-tag">
+          <span class="red-bg">FEATURES</span><span class="mag-trans">NEWS</span>
+        </div>
+        <div class="card-body card-body-pad">
+          <h5 class="card-title fam-hd">
+            Florida Dem Rep Shalala Admits Squad Sometimes Pisses Her Off,
+            Lets Loose on 'Idiotic' Trump
+          </h5>
+        </div>
+      </div>
+      <div class="card col-md-3">
+        <div class="mag-img-div">
+          <img class="w-100 h-100 mag-img" src="./images/ft4.jpg" alt="Card image cap" />
+        </div>
+        <div class="mag-tag">
+          <span class="red-bg">PERISCOPE</span><span class="mag-trans">POLITICS</span>
+        </div>
+        <div class="card-body card-body-pad">
+          <h5 class="card-title fam-hd">
+            Florida Dem Rep Shalala Admits Squad Sometimes Pisses Her Off,
+            Lets Loose on 'Idiotic' Trump
+          </h5>
+        </div>
+      </div>
+      <div class="card col-md-3">
+        <div class="mag-img-div">
+          <img class="w-100 h-100 mag-img" src="./images/ft5.jpg" alt="Card image cap" />
+        </div>
+        <div class="mag-tag">
+          <span class="red-bg">FEATURES</span><span class="mag-trans">CULTURE</span>
+        </div>
+        <div class="card-body card-body-pad">
+          <h5 class="card-title fam-hd">
+            Here Are the Young, Diverse Conservative Stars Progressives Need
+            to Fear
+          </h5>
+        </div>
+      </div>
+      <div class="card col-md-3">
+        <div class="mag-img-div">
+          <img class="w-100 h-100 mag-img" src="./images/ft6.jpg" alt="Card image cap" />
+        </div>
+        <div class="mag-tag">
+          <span class="red-bg">POLITICS</span><span class="mag-trans">NEWS</span>
+        </div>
+        <div class="card-body card-body-pad">
+          <h5 class="card-title fam-hd">
+            Janelle Monáe's Film 'Antebellum' Shows the Horrors of Systemic
+            Racism
+          </h5>
+        </div>
+      </div>
+      <div class="card col-md-3">
+        <div class="mag-img-div">
+          <img class="w-100 h-100 mag-img" src="./images/ft7.jpg" alt="Card image cap" />
+        </div>
+        <div class="mag-tag">
+          <span class="red-bg">FOCUS</span><span class="mag-trans">CULTURE</span>
+        </div>
+        <div class="card-body card-body-pad">
+          <h5 class="card-title fam-hd">
+            Dr. Fauci Would Bet 10 Cents on Trump Having Vaccine by November,
+            December
+          </h5>
+        </div>
+      </div>
+      <!-- </div> -->
+    </div>
+    <div class="row align-items-center justify-content-center section-topic">
+      <!-- <div class="col-"> -->
+      <span class="n-log">N</span>
+      <h2 class="topic-h2 mb-0">EDITORS PICK</h2>
+      <!-- </div> -->
```

Comment:
- [ ] Please delete **commented piece of codes** in your HTML and CSS files. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))  
:x: **commented piece of codes** /* `.nav { color: red;}` */  
:heavy_check_mark: **comments about code** /*Navigation */